SOOP/SOOP_ASF_SST_XBT/soop_xbt_pipeline/create_plot.py

```diff
@@ -5,25 +5,27 @@
 # Institute: IMOS / eMII
 # email address: laurent.besnard@utas.edu.au
 
-import sys, os
-import tempfile
+import datetime
+import itertools
+import os
+import re
 import shutil
+import string
+import sys
+import tempfile
+
+import matplotlib
+import matplotlib.pyplot as plt
+import pylab
+from matplotlib.pyplot import figure, plot, show, title, xlabel, xticks, ylabel
+from netCDF4 import Dataset, num2date
+from numpy import ma
 
 # the next 3 lines are needed to create plots without a xserver
 MPLCONFIGDIR = tempfile.mkdtemp()
 os.environ['MPLCONFIGDIR'] = MPLCONFIGDIR
-import matplotlib
```

Comment:
If I recall correctly, this was required for the older version of matplotlib, and should now be redundant with the newer one? https://github.com/aodn/data-services/pull/581